core/sdk-core/src/test/java/software/amazon/awssdk/core/async/ChunkBufferTest.java

```diff
@@ -35,11 +35,6 @@
 
 class ChunkBufferTest {
```

Comment:
Shouldn't we have unit tests for the new capability of the ChunkBuffer (ie unknown length)?